bin/docker-run

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+trap "exit" INT
+
+DOCKER_IMAGE_NAME="nextstrain/ncov-ingest"
+DOCKER_CONTAINER_NAME="${DOCKER_IMAGE_NAME//\//-}-$(TZ=UTC date "+%Y-%m-%d--%H-%M-%S--%Z")"
+
+GITHUB_BRANCH=$(git rev-parse --abbrev-ref HEAD) || "unknown"
+
+declare -a config
+config+=(
+  database_name="gisaid"
+  s3_src="s3://nextstrain-ncov-private/branch/${GITHUB_BRANCH}"
+  s3_dst="s3://nextstrain-ncov-private/branch/${GITHUB_BRANCH}"
+  fetch_from_database=false
+  trigger_preprocessing=false
+  keep_all_files: true
+)
```

Comment:
Note that, as always, because it interacts with prod data, it's extremely easy to mess things up in production, by just setting a wrong S3 url here. So be careful.